tests/ui-cargo/module_style/fail_mod/src/main.stderr

```diff
@@ -4,7 +4,7 @@ error: `mod.rs` files are required, found `/bad/inner.rs`
 LL | pub mod stuff;
    | ^
    |
-   = note: `-D clippy::self-named-mod-files` implied by `-D warnings`
+   = note: `-D clippy::self-named-module-files` implied by `-D warnings`
    = help: create a `/bad/inner/mod.rs` file and remove `/bad/inner.rs`
```

Comment:
Should the suggestion simple be "move `bad/inner.rs` to `bad/inner/mod.rs`, rather than "create", "remove"?

Reply:
I changed both error messages to the above suggestion